cmd/bee/cmd/cmd.go

```diff
@@ -61,6 +61,7 @@ const (
 	optionNameSwapInitialDeposit         = "swap-initial-deposit"
 	optionNameSwapEnable                 = "swap-enable"
 	optionNameChequebookEnable           = "chequebook-enable"
+	optionNameStubSwapChain              = "stub-swap-chain"
```

Comment:
maybe `no-chain` or `no-backend` or `no-blockchain`, `noop-blockchain` etc... maybe @agazso can have some input here what would be some reasonable cli flag to toggle this feature